assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/AnalyticsStats.js

```diff
@@ -85,6 +88,12 @@ const AnalyticsStats = ( {
 		},
 	};
 
+	if ( isZeroReport( data ) ) {
```

Comment:
This doesn't work as expected because `data` includes metrics for both goals _and_ bounce rate (which is shown in place of goals on the entity dashboard). `isZeroReport` is only true if _all_ the report data is zero, which means that it will return false if the bounce rate is non-zero.

We need to handle things a bit differently based on which stat is selected (goals or bounce rate) which can be determined by the `selectedStats` prop:

![image](https://user-images.githubusercontent.com/1621608/151939863-dfd8a0ff-4661-4384-8060-6342f1cf2c6c.png)
![image](https://user-images.githubusercontent.com/1621608/151939867-5cdbb0b7-92af-4b2c-a758-cee7ed47a02e.png)

Reply:
Ah, right on, makes sense. 👍🏻